### ISSUE TITLE
Cannot install `@fedify/cli` in npm in arm64 Windows 

### ISSUE THREAD

**Issue by saschanaz** [dahlia/fedify#160]
```
> npm i -g @fedify/cli
npm error code 1
npm error path C:\Users\sasch\AppData\Roaming\npm\node_modules\@fedify\cli
npm error command failed
npm error command C:\Windows\system32\cmd.exe /d /s /c node install.mjs
npm error Unsupported architecture: arm64
npm error node:internal/deps/undici/undici:13178
npm error       Error.captureStackTrace(err);
npm error             ^
npm error
npm error TypeError: Failed to parse URL from undefined
npm error     at node:internal/deps/undici/undici:13178:13
npm error     at async download (file:///C:/Users/sasch/AppData/Roaming/npm/node_modules/@fedify/cli/install.mjs:87:20)
npm error     at async install (file:///C:/Users/sasch/AppData/Roaming/npm/node_modules/@fedify/cli/install.mjs:46:24)
npm error     at async main (file:///C:/Users/sasch/AppData/Roaming/npm/node_modules/@fedify/cli/install.mjs:41:3)
npm error     at async file:///C:/Users/sasch/AppData/Roaming/npm/node_modules/@fedify/cli/install.mjs:176:3 {
npm error   [cause]: TypeError: Invalid URL
npm error       at new URL (node:internal/url:797:36)
npm error       at new Request (node:internal/deps/undici/undici:9269:25)
npm error       at fetch (node:internal/deps/undici/undici:9998:25)
npm error       at fetch (node:internal/deps/undici/undici:13176:10)
npm error       at fetch (node:internal/bootstrap/web/exposed-window-or-worker:72:12)
npm error       at download (file:///C:/Users/sasch/AppData/Roaming/npm/node_modules/@fedify/cli/install.mjs:87:26)
npm error       at install (file:///C:/Users/sasch/AppData/Roaming/npm/node_modules/@fedify/cli/install.mjs:46:30)
npm error       at main (file:///C:/Users/sasch/AppData/Roaming/npm/node_modules/@fedify/cli/install.mjs:41:9)
npm error       at async file:///C:/Users/sasch/AppData/Roaming/npm/node_modules/@fedify/cli/install.mjs:176:3 {
npm error     code: 'ERR_INVALID_URL',
npm error     input: 'undefined'
npm error   }
npm error }
npm error
npm error Node.js v20.17.0
npm error A complete log of this run can be found in: C:\Users\sasch\AppData\Local\npm-cache\_logs\2024-10-31T19_06_49_791Z-debug-0.log
```

Probably can fall back to x86-64 until there is arm64 builder in Actions?

**Comment by ThisIsMissEm** [comment]
This cannot be fixed as deno does not support compile for ARM64 windows: https://docs.deno.com/runtime/reference/cli/compiler/#cross-compilation

**Comment by ThisIsMissEm** [comment]
See: https://github.com/denoland/deno/issues/13331

**Comment by saschanaz** [comment]
I mean, given arm64 Windows can run x86-64 via emulation, maybe that can be downloaded instead?

**Comment by ThisIsMissEm** [comment]
If that's the case, then I guess it could be added here: https://github.com/dahlia/fedify/blob/main/cli/npm/install.mjs#L27-L29

**Comment by dahlia** [comment]
Fixed 1.2.2. Try `npm install -g @fedify/cli@1.2.2`.
